source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import LinkIcon from 'octicon/link.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+function repoListInit(): void {
+	for (const repo of select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]')) {
+		repo.after(
+			<a
+				href={`https://${repo.textContent!.trim()}`}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{' '}<LinkExternalIcon/>
```

Comment:
Also aligns it better
```suggestion
			>
				<LinkExternalIcon className="v-align-middle"/>
```